production_configs/20220303_tuned_ze40_south/readme.md

```diff
@@ -0,0 +1,11 @@
+# Ze40 south pointing Config
+
+## 20220304_zen40az180_dl1ab_tuned_nsb_psf
```

Comment:
Remove the date, as it is not essential or provides any relevant information to the actual production